clc/config/wizard_provider.go

```diff
@@ -13,40 +13,40 @@ import (
 	"github.com/hazelcast/hazelcast-go-client"
 	"github.com/spf13/pflag"
 
-	"github.com/hazelcast/hazelcast-commandline-client/clc/config"
+	"github.com/hazelcast/hazelcast-commandline-client/clc/config/wizard"
 	"github.com/hazelcast/hazelcast-commandline-client/clc/paths"
 	clcerrors "github.com/hazelcast/hazelcast-commandline-client/errors"
 	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
 )
 
-type Provider struct {
-	fp  *atomic.Pointer[config.FileProvider]
+type WizardProvider struct {
```

Comment:
Is it necessary to move this provider from `clc/config/wizard`?

Reply:
It is not necessary but I believe it is simpler to have all providers in the same package scope.